plugins/main/bridge/bridge.go

```diff
@@ -41,7 +42,10 @@ import (
 )
 
 // For testcases to force an error after IPAM has been performed
-var debugPostIPAMError error
+var (
+	debugPostIPAMError error
+	logger             = log.New(os.Stdout, "", log.Ldate|log.Ltime)
```

Comment:
Stdout is used as part of the response protocol, so we can't log to it. We need to use stderr.

Reply:
I was a little confused about it, and the absence of logs didn't affect functionality, so I deleted it.